src/Data/Array.purs

```diff
@@ -551,6 +553,31 @@ alterAt i f xs = maybe Nothing go (xs !! i)
 -- Transformations -------------------------------------------------------------
 --------------------------------------------------------------------------------
 
+-- | Inserts the given element in between each element in the array. The array
+-- | must have two or more elements for this operation to take effect.
+-- |
+-- | ```purescript
+-- | intersperse " " [ "a", "b" ] == [ "a", " ", "b" ]
+-- | intersperse 0 [ 1, 2, 3, 4, 5 ] == [ 1, 0, 2, 0, 3, 0, 4, 0, 5 ]
+-- | ```
+-- |
+-- | If the array has one or zero elements, the input array is returned.
```

Comment:
I think "less than two" might read a bit more naturally here? Up to you.

Reply:
I don't know. I think I could go either way. 
Meh, let's just use the "less than two."